### ISSUE TITLE
autocomplete in mathmode does not work on ST4

### ISSUE THREAD

**Issue by klepo0** [SublimeText/LaTeXTools#1524]
Hello, 

There are already a couple posts on autocompletion on ST4 but the answers there are not working on my system. 
Here is the issue: inside a math environment, I usually type "a" then tab to get "\alpha". But on ST4, the autocomplete just shows me a list of possible options. For writing an article with many Greek symbols, it is not practical to scroll down the list every time. 

Other key-bindings also does not work: "ba" then tab would give me 
\begin{align*}

\end{align*}
on ST3 but on ST4 it will give me other completely different options. 
I am not a tech guru, but will be able to give you further information from my system if it is needed.

**Comment by msiniscalchi** [comment]
Yeah... the new-style autocomplete functionality seems to trump LaTeXTools' expansion. Not sure what's happening with ba; it seems to be shadowed by e.g. baa (begin align at).

**Comment by wahspilihp** [comment]
Something seems to be eating the leading slash on autocompleted latex commands too. I don't know if it is one of the other packages I have installed conflicting with LatexTools or if it was the new completion system (or both), but it only started happening when I updated to ST4.

**Comment by jiafengkevinchen** [comment]
One potential option is to disable tab_completion, but doing so makes tab only respond to _snippets_, and for some reason stuff like `bal` isn't considered a snippet---seems like the reason is that these things are contained in .sublime-completions rather than .sublime-snippet...

I also posted this on the ST forum: 
https://forum.sublimetext.com/t/treating-sublime-completions-as-snippets-when-tab-completion-is-turned-off/58437?u=jchen

**Comment by ishfulness** [comment]
> Something seems to be eating the leading slash on autocompleted latex commands too. I don't know if it is one of the other packages I have installed conflicting with LatexTools or if it was the new completion system (or both), but it only started happening when I updated to ST4.

At least for the backslash issue (which looks to be related to the latex-cwl package), there is a pull request now with a fix.
https://github.com/SublimeText/LaTeXTools/pull/1454/files

I changed the line in my latex_cwl_completions.py file manually and it now seems to work

**Comment by ankitp94** [comment]
> One potential option is to disable tab_completion, but doing so makes tab only respond to _snippets_, and for some reason stuff like `bal` isn't considered a snippet---seems like the reason is that these things are contained in .sublime-completions rather than .sublime-snippet...
> 
> I also posted this on the ST forum: https://forum.sublimetext.com/t/treating-sublime-completions-as-snippets-when-tab-completion-is-turned-off/58437?u=jchen

Did you find a fix?

**Comment by deathaxe** [comment]
> Here is the issue: inside a math environment, I usually type "a" then tab to get "\alpha". But on ST4, the autocomplete just shows me a list of possible options. For writing an article with many Greek symbols, it is not practical to scroll down the list every time.

This is caused by "a" being implemented as "sublime-completion" rather than "sublime-snippet" file. ST4 drops all completions which perfectly match typed prefix.
